src/components/PhotoUpload/PhotoUploadCaptchaToolbar.js

```diff
@@ -90,7 +96,7 @@ const StyledToolbar = styled(Toolbar)`
     }
   }
 
-  > header {
+  header {
```

Comment:
This could override styles accidentally as there might be headers at deeper nesting levels – could this be more specific?

Reply:
Will not change
- there are no headers below, also nesting headers in headers seems a bit too much.
- this is very deep in our component hierarchy, no further rabbit hole planned for now
- all the code is confined in one file
- child selector make refactorings a nightmare, changing hierarchy needs css adaptation